packages/react/src/components/Header/Header.jsx

```diff
@@ -77,8 +77,9 @@ const propTypes = {
     openMenu: PropTypes.string,
     closeMenu: PropTypes.string,
   }),
-
   testId: PropTypes.string,
+  /** Returns true, if the icon should be shown. (itemLabel) => {} */
```

Comment:
Pass the entire item action to the isItemVisible and then allow action items to have an .id field.  We cannot use the label since this would be too hard to manage from a localization point of view.  ie, we if want to hide the 'logout' action we can test for item.id === 'logout' which will be the same no matter what language we are in.

Reply:
@stuckless, Good call. I've made these adjustments and added ids to the default action items in SuiteHeader.  https://github.com/carbon-design-system/carbon-addons-iot-react/pull/2768/commits/34550b6eac2181711029b982f512b41ac53c2283